omnipaxos/src/storage.rs

```diff
@@ -309,6 +309,13 @@ where
         }
     }
 
+    pub(crate) fn rollback_and_panic(&mut self, values: Vec<RollbackValue>, msg: &str) {
```

Comment:
Let's use this function when possible, for example [here](https://github.com/haraldng/omnipaxos/blob/c883cc9c25d03f047820c016d761f6570b8b4404/omnipaxos/src/sequence_paxos/follower.rs#L259) and [here](https://github.com/haraldng/omnipaxos/blob/c883cc9c25d03f047820c016d761f6570b8b4404/omnipaxos/src/sequence_paxos/follower.rs#L259). (Please check other places if we can replace it)